security/cmd/node_agent_k8s/nodeagent.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: nodeagent
+spec:
+  template:
+    metadata:
+      labels:
+        app: nodeagent
+        version: v1
+    spec:
+      initContainers:
+      - name: flexvol-driver
+        image: quay.io/saurabh/flexvol:latest
```

Comment:
what is "quay"?

Reply:
it's some repo like gcr, replaced with gcr.io now